mods/digimon/pokedex.js

```diff
@@ -2370,4 +2370,4 @@ let BattlePokedex = {
 	},
 };
 
-exports.BattlePokedex = BattlePokedex;
+exports.DG-NormalPokedex = DG-NormalPokedex;
```

Comment:
When you plug this into PS as a mod, PS automatically imports Module#BattlePokedex, and not any other property of ``module.exports``.

Reply:
Its an error from auto-replace sorry. I will be closing the PR on this. :)